### ISSUE TITLE
Mac GUI works!  

### ISSUE THREAD

**Issue by sjackwu** [Winetricks/winetricks#1148]
This is not a bug report, but rather a howto which I can't find on the Internet. Please consider putting this into your wiki so that people know how to get GUI on Mac.

For a long time I thought winetricks doesn't have a winetricks GUI on Mac OS, but today I figure out a way to launch the GUI.

I use homebrew to install everything and the dependencies: wine, winetricks and zenity.

And then the magic words, enter this command in the Mac Terminal.app:
     DISPLAY=zenity winetricks

That's it! 

![item-0_ _menubar](https://user-images.githubusercontent.com/17001440/49511436-1e474380-f8c6-11e8-8e5f-a87c1d52734c.png)

**Comment by sjackwu** [comment]
On a second thought, can we make the `--gui` argument works not only as a diagnostics but also a GUI launcher?

1. if no zenity is installed, hint user to install it
1. if zenity is installed, launch the GUI directly.

I think this will help Mac users to have the GUI.

**Comment by austin987** [comment]
DISPLAY is an X11 variable, it's not meant for setting the gui.

We already tell users what to do if --gui fails:
https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L3231

improvements are welcome, send a PR if you have suggestions.

**Comment by sjackwu** [comment]
@austin987 thank you for responding!

I did see the comments in line 3231 and did follow the instruction to install zenity, but when I tried to launch winetricks I saw the following message:

> warning: taskset/cpuset not available on your platform!
>
> You are using a 64-bit WINEPREFIX. Note that many verbs only install 32-bit versions of packages. If you encounter problems, please retest in a clean 32-bit WINEPREFIX before reporting a bug.
>
> Using winetricks 20181203 - sha256sum: 2e32c987b5dd0ddd3e0836290d9561e7f251ca080248f53050cc84c27d4bbd18 with wine-3.0.4 and WINEARCH=win64
>
> DISPLAY not set, not defaulting to gui

It was the last line "*DISPLAY not set, not defaulting to gui*" which triggered me to use `DISPLAY=zenity winetricks` to launch winetricks GUI.

BTW, in case you're wondering, I didn't install any X11, e.g. xquartz.

> DISPLAY is an X11 variable, it's not meant for setting the gui.
> 
> We already tell users what to do if --gui fails:
> https://github.com/Winetricks/winetricks/blob/master/src/winetricks#L3231
> 
> improvements are welcome, send a PR if you have suggestions.

**Comment by austin987** [comment]
Makes sense, we probably don't handle the wine mac driver properly.

**Comment by austin987** [comment]
@sjackwu so setting DISPLAY works by chance. You could also set it to DISPLAY=foobar, and zenity would've also worked.

The issue was that DISPLAY is only set if XQuartz is running. If not, wine will use the mac driver, but winetricks assumes you're running in a console with no GUI, so disables it.

I've made a patch that fixes that for mac, so it still tries to get a gui. The branch is here:
https://github.com/austin987/winetricks/tree/osx-display

winetricks itself is:
https://raw.githubusercontent.com/austin987/winetricks/osx-display/src/winetricks

please lmk if that works for you (i.e., unset DISPLAY and retry)

**Comment by austin987** [comment]
@sjackwu no, that was 100% my fault. I wrote and tested a patch earlier, then just rewrote and stupidly didn't test it.

Retry please, I fixed the mistake. Works for me on linux (if I swap Darwin/Linux).

**Comment by sjackwu** [comment]
@austin987 the new version works as expected and shows a meesage below:
>Running on OSX, but DISPLAY is not set...probably using Mac Driver.
>winetricks GUI enabled, using zenity 3.30.0

Thank you!
